docs/index.rst

```diff
@@ -1,14 +1,162 @@
-Welcome to Betelgeuse's documentation!
-======================================
+Betelgeuse documentation
+========================
 
-Betelgeuse reads standard Python test cases and offers tools to interact with
-Polarion. Possible interactions are:
+.. contents:: Topics
 
-* Automatic creation and update of Requirements and Test Cases from a Python
+.. _what_is_betelgeuse:
+
+What is Betelgeuse?
+```````````````````
+
+Betelgeuse is a python program that reads standard Python test cases and offers
+tools to interact with Polarion. Possible interactions are:
+
+* Automatic creation/updation of Requirements and Test Cases from a Python
   project code base.
 * Creation of Test Runs based on a jUnit XML file.
```

Comment:
This should also state that it needs the source code as well in order to proper map the ID.